HIRS_ProvisionerTPM2/CMakeLists.txt

```diff
@@ -159,13 +159,25 @@ if(CMAKE_THREAD_LIBS_INIT)
 endif()
 
 # Set commands for installation of project on target system (i.e. "make install")
-install(TARGETS ${PROJECT_NAME}
+install(TARGETS ${PROJECT_NAME} tpm_version
         DESTINATION "bin")
 install(FILES config/log4cplus_config.ini DESTINATION /etc/hirs/TPM2_Provisioner)
 install(FILES scripts/tpm_aca_provision DESTINATION /usr/local/bin PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 install(DIRECTORY DESTINATION ${/var/log/hirs/provisioner}
         DIRECTORY_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
                               GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+
+install(DIRECTORY DESTINATION ${/etc/hirs/provisioner}
+        DIRECTORY_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
+                              GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+install(FILES ../HIRS_Provisioner/src/main/resources/defaults.properties DESTINATION /etc/hirs/provisioner RENAME provisioner.properties)
+install(FILES ../HIRS_Provisioner/hirs-provisioner-config.sh DESTINATION /etc/hirs/provisioner
```

Comment:
I'm guessing this has to do with the other projects that aren't touched on in this repo?

Reply:
I believe the client work requires the provisioner.properties config. This is just updating the 2.0 provisioner to drop that file on the machine because it is currently only installed by the 1.2 provisioner.

Reply:
marking resolved